### PR TITLE
File Added : Assignment_5 and Assignment_6 and Assignment_7

### PR DESCRIPTION
**Pull request by VIP5614**

### CODE REVIEW

**Review by pratikshaghule — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

**Review by avinash0070 — CHANGES_REQUESTED** [review]
Change title to File Added : Assignment_5 and Assignment_6

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <body style="text-align: center;">
+    <h1>Fibonacci Series Upto Given Number</h1> <hr>
+        <label for="number">Enter a Number: </label>
+        <input type="text" id="number" placeholder="Enter a Number='5'">
+        <br>
+        <button type="submit" onclick="fibonacciSeries()">Generate Fibonacci Series</button>
+        </form>
+        <p id="result" style="display: none;"><b>Fibonacci Series:</b> </p>
+  </body>
+</html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html:35]: bad code indentation need 4 space

File: Vish/JS/Assignment-5/Sum-match.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      }
+    </script>
+  </head>
+  <body style="text-align: center;">
+    <h2>Sum Match</h2> <hr>
+    Enter 3 digit Number: <input type="text" id="no" />
+    <button onclick="cal()">calculate</button>
+    <p id="p"></p>
+  </body>
+</html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Sum-match.html:33]: Add new line

File: Vish/JS/Assignment-5/Sum-match.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          document.getElementById("p").innerHTML =
+            "Invalid Number!!Please enter a 3 digit no.";
+      }
+    </script>
+  </head>
+  <body style="text-align: center;">
+    <h2>Sum Match</h2> <hr>
+    Enter 3 digit Number: <input type="text" id="no" />
+    <button onclick="cal()">calculate</button>
+    <p id="p"></p>
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Sum-match.html]: give proper name to this id

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    <title>Document</title>
+    <script>
+        function fibonacciSeries() {
+          event.preventDefault();
+  
+          let numbers = parseInt(document.getElementById("number").value);
+          let result = document.getElementById("result");
+  
+          let firstNumber = 0,
+            secondNumber = 1;
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: where is declare  secondNumber variable

File: Vish/JS/Assignment-5/Sum-match.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script>
+      function cal() {
+        let n = document.getElementById("no").value;
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Sum-match.html]: give proper name to this variable

File: Vish/JS/Assignment-5/Sum-match.html [hunk 6]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Sum-match.html:1]: bad code indentation

File: Vish/JS/Assignment-6/list-of-name.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script>
+        function enterNamesList() {
+                let namesList = document.getElementById('namesList');
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/list-of-name.html:9]: use 4 space only

File: Vish/JS/Assignment-6/string.html [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+            <input type="text" id="val_ten"><br><br>
+            <label for="nword">Search word:</label>
+            <input type="text" id="nword"><br><br>
+            <label for="rword">Enter replace word:</label>
+            <input type="text" id="rword"><br>
+            <button type="button" onclick="replace_str()">Submit</button>
+            <h4 id="res_ten"></h4>
+        </form>
+    </body>
+</html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html:170]: add new line

File: Vish/JS/Assignment-6/sumFactorial.html [hunk 9]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/sumFactorial.html:1]: need proper code indentation

File: Vish/JS/Assignment-7/words-and-whitespaces.html [hunk 10]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-7/words-and-whitespaces.html:1]: bad code indentation

**Review by avinash0070 — CHANGES_REQUESTED** [review]
assign yourself

**Review by avinash0070 — COMMENTED** [review]
need proper code indentation to all files

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script>
+        function fibonacciSeries() {
+          event.preventDefault();
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script>
+        function fibonacciSeries() {
+          event.preventDefault();
+  
+          let numbers = parseInt(document.getElementById("number").value);
+          let result = document.getElementById("result");
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+        function fibonacciSeries() {
+          event.preventDefault();
+  
+          let numbers = parseInt(document.getElementById("number").value);
+          let result = document.getElementById("result");
+  
+          let firstNumber = 0,
+            secondNumber = 1;
+          let nextNumber = 0;
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+  
+          let numbers = parseInt(document.getElementById("number").value);
+          let result = document.getElementById("result");
+  
+          let firstNumber = 0,
+            secondNumber = 1;
+          let nextNumber = 0;
+  
+          result.style.display = "block";
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extr line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+  
+          let firstNumber = 0,
+            secondNumber = 1;
+          let nextNumber = 0;
+  
+          result.style.display = "block";
+  
+          for (let number = 1; number <= numbers; number++) {
+            result.innerHTML += `<br> ${firstNumber}`;
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-6/list-of-name.html [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+                let namesListNewLine = document.getElementById('namesListNewLine');
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/list-of-name.html]: remove extra line

File: Vish/JS/Assignment-6/list-of-name.html [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+                let listOfNames = [];
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/list-of-name.html]: Remove extra line

File: Vish/JS/Assignment-6/list-of-name.html [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+                });
+            }
+    </script>
+    <style>
+      h4 {
+        color: rgb(197, 26, 26);
+      }
+      /* span {
+        color: rgb(134, 65, 161);
+      } */
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/list-of-name.html]: Remove commented code

File: Vish/JS/Assignment-7/Number_Palindrom.html [hunk 19]
```diff
@@ -0,0 +1,36 @@
+<html>
+	<head>
+        <title>Number Palindrome</title>
+		<script>
+			function Palindrome()
+			{
+				var rem, temp, final = 0;
+				var number = Number(document.getElementById("N").value);
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-7/Number_Palindrom.html]: remove extra line

File: Vish/JS/Assignment-7/Number_Palindrom.html [hunk 20]
```diff
@@ -0,0 +1,36 @@
+<html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-7/Number_Palindrom.html:1]: need proper code indentation

File: Vish/JS/Assignment-7/words-and-whitespaces.html [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+      }
+    </script>
+  </head>
+  <body>
+    <br />
+    <h1>Words and Whitespaces</h1>
+    Enter The String :<input type="text" name="n" id="N" />
+    <hr color="cyan" />
+    <br />
+    <center><button onClick="WordCount()">CHECK</button></center>
```
Comment (avinash0070) [inline Vish/JS/Assignment-7/words-and-whitespaces.html]: change CHECK to Check

File: Vish/JS/Assignment-6/sumNumber.html [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script>
+      function sumOfNumber() {
+        let total = document.getElementById("total");
+
+        let numbers = [];
+
+        document.querySelector("h4").style.display = "none";
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/sumNumber.html]: Remove extra line

File: Vish/JS/Assignment-6/sumNumber.html [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script>
+      function sumOfNumber() {
+        let total = document.getElementById("total");
+
+        let numbers = [];
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/sumNumber.html]: Remove extra line

File: Vish/JS/Assignment-6/sumNumber.html [hunk 24]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script>
+      function sumOfNumber() {
+        let total = document.getElementById("total");
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/sumNumber.html]: Remove extra line

File: Vish/JS/Assignment-5/Sum-match.html (hunk 2 above)
Comment (VIP5614) [reply Vish/JS/Assignment-5/Sum-match.html:33]: done

File: Vish/JS/Assignment-5/Fibonacci-Series.html (hunk 4 above)
Comment (VIP5614) [reply Vish/JS/Assignment-5/Fibonacci-Series.html]: done

File: Vish/JS/Assignment-5/Sum-match.html (hunk 3 above)
Comment (VIP5614) [reply Vish/JS/Assignment-5/Sum-match.html]: done

File: Vish/JS/Assignment-5/Sum-match.html (hunk 6 above)
Comment (VIP5614) [reply Vish/JS/Assignment-5/Sum-match.html:1]: done

File: Vish/JS/Assignment-6/list-of-name.html (hunk 7 above)
Comment (VIP5614) [reply Vish/JS/Assignment-6/list-of-name.html:9]: done

File: Vish/JS/Assignment-6/string.html (hunk 8 above)
Comment (VIP5614) [reply Vish/JS/Assignment-6/string.html:170]: done

File: Vish/JS/Assignment-6/sumFactorial.html (hunk 9 above)
Comment (VIP5614) [reply Vish/JS/Assignment-6/sumFactorial.html:1]: done

File: Vish/JS/Assignment-7/words-and-whitespaces.html (hunk 10 above)
Comment (VIP5614) [reply Vish/JS/Assignment-7/words-and-whitespaces.html:1]: proper code indentation done

File: Vish/JS/Assignment-5/Sum-match.html [hunk 25]
```diff
@@ -26,6 +26,7 @@ <h2>Sum Match</h2> <hr>
         <label> 3 digit Number: </label>> 
         <input type="text" id="no" />
         <button onclick="cal()">calculate</button>
+        <br>
         <p id="demo"></p>
     </body>
 </html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Sum-match.html:33]: Add new line

File: Vish/JS/Assignment-6/string.html [hunk 26]
```diff
@@ -83,37 +83,47 @@ <h3>Trim string</h3>
...
             <h4 id="res_one"></h4>
+
             <!--length-->
             <h3>String length</h3>
             <label for="val_two">Enter text:</label>
             <input type="text" id="val_two"><br>
             <button type="button" onclick="strlength()">Submit</button>
+            <br>
             <h4 id="res_two"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 27]
```diff
@@ -83,37 +83,47 @@ <h3>Trim string</h3>
...
             <!--last index-->
             <h3>Last index</h3>
             <label for="val_four">Enter text:</label>
             <input type="text" id="val_four"><br><br>
             <label for="input_index">Find last index of:</label>
             <input type="text" id="input_index"><br>
             <button type="button" onclick="lastindex()">Submit</button>
+            <br>
             <h4 id="res_four"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 28]
```diff
@@ -123,27 +133,35 @@ <h3>Slice</h3>
             <label for="l_index">Last index:</label>
             <input type="number" id="l_index"><br>
             <button type="button" onclick="slicestr()">Submit</button>
+            <br>
             <h4 id="res_six"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 29]
```diff
@@ -123,27 +133,35 @@ <h3>Slice</h3>
...
             <h4 id="res_seven"></h4>
+            
             <!--uppercase-->
             <h3>Upper case</h3>
             <label for="val_eight">Enter text:</label>
             <input type="text" id="val_eight"><br>
             <button type="button" onclick="upper_case()">Submit</button>
+            <br>
             <h4 id="res_eight"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: Remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 30]
```diff
@@ -153,7 +171,9 @@ <h3>Replace</h3>
             <label for="rword">Enter replace word:</label>
             <input type="text" id="rword"><br>
             <button type="button" onclick="replace_str()">Submit</button>
+            <br>
             <h4 id="res_ten"></h4>
+        
         </form>
     </body>
 </html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html:170]: add new line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 31]
```diff
@@ -0,0 +1,40 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+            <script>
+                function fibonacciSeries() {
+                event.preventDefault();
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 32]
```diff
@@ -0,0 +1,40 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+            <script>
+                function fibonacciSeries() {
+                event.preventDefault();
+        
+                let numbers = parseInt(document.getElementById("number").value);
+                let result = document.getElementById("result");
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 33]
```diff
@@ -0,0 +1,40 @@
...
+        
+                let numbers = parseInt(document.getElementById("number").value);
+                let result = document.getElementById("result");
+        
+                let firstNumber = 0;
+                let secondNumber = 1;
+                let nextNumber = 0;
+        
+                result.style.display = "block";
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 34]
```diff
@@ -0,0 +1,40 @@
...
+                function fibonacciSeries() {
+                event.preventDefault();
+        
+                let numbers = parseInt(document.getElementById("number").value);
+                let result = document.getElementById("result");
+        
+                let firstNumber = 0;
+                let secondNumber = 1;
+                let nextNumber = 0;
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: remove extra line

File: Vish/JS/Assignment-5/Fibonacci-Series.html [hunk 35]
```diff
@@ -0,0 +1,40 @@
...
+        
+                let firstNumber = 0;
+                let secondNumber = 1;
+                let nextNumber = 0;
+        
+                result.style.display = "block";
+        
+                for (let number = 1; number <= numbers; number++) {
+                  result.innerHTML += `<br> ${firstNumber}`;
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-5/Fibonacci-Series.html]: Remove this extra line

File: Vish/JS/Assignment-6/everyNumberFactorial.html [hunk 36]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/everyNumberFactorial.html:1]: add proper code indentation

File: Vish/JS/Assignment-6/list-of-name-Ascending&Descending.html [hunk 37]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/list-of-name-Ascending&Descending.html:1]: need proper code indatation

File: Vish/JS/Assignment-6/string.html [hunk 38]
```diff
@@ -0,0 +1,179 @@
...
+            <!--concat-->
+            <h3>Concat</h3>
+            <label for="val_five">Enter first text:</label>
+            <input type="text" id="val_five"><br><br>
+            <label for="val_addfive">Enter second text:</label>
+            <input type="text" id="val_addfive"><br>
+            <button type="button" onclick="concatstr()">Submit</button>
+            <br>
+            <h4 id="res_five"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: Remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 39]
```diff
@@ -0,0 +1,179 @@
...
+            <h4 id="res_six"></h4>
+            
+            <!--lowercase-->
+            <h3>Lower case</h3>
+            <label for="val_seven">Enter text:</label>
+            <input type="text" id="val_seven"><br>
+            <button type="button" onclick="lower_case()">Submit</button>
+            <br>
+            <h4 id="res_seven"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 40]
```diff
@@ -0,0 +1,179 @@
...
+            <!--search-->
+            <h3>Search string</h3>
+            <label for="val_nine">Enter text:</label>
+            <input type="text" id="val_nine"><br><br>
+            <label for="inputword">Search word:</label>
+            <input type="text" id="inputword"><br>
+            <button type="button" onclick="search_str()">Submit</button>
+            <br>
+            <h4 id="res_nine"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: remove extra line

File: Vish/JS/Assignment-6/string.html [hunk 41]
```diff
@@ -0,0 +1,179 @@
...
+            <label for="val_ten">Enter text:</label>
+            <input type="text" id="val_ten"><br><br>
+            <label for="nword">Search word:</label>
+            <input type="text" id="nword"><br><br>
+            <label for="rword">Enter replace word:</label>
+            <input type="text" id="rword"><br>
+            <button type="button" onclick="replace_str()">Submit</button>
+            <br>
+            <h4 id="res_ten"></h4>
+
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html]: remove extra line

**Review by avinash0070 — CHANGES_REQUESTED** [review]

File: Vish/JS/Assignment-6/string.html [hunk 42]
```diff
@@ -173,7 +164,6 @@ <h3>Replace</h3>
             <button type="button" onclick="replace_str()">Submit</button>
             <br>
             <h4 id="res_ten"></h4>
-        
         </form>
     </body>
 </html>
```
Comment (avinash0070) [inline Vish/JS/Assignment-6/string.html:170]: add new line

**Review by avinash0070 — CHANGES_REQUESTED** [review]

File: Vish/JS/Assignment-6/string.html (hunk 42 above)
Comment (VIP5614) [reply Vish/JS/Assignment-6/string.html:170]: Done

**Review by avinash0070 — APPROVED** [review]

**Pull request merged**